gramdictru/Scripts/Utils/copyUtils.tsx

```diff
@@ -0,0 +1,273 @@
+﻿import * as React from "react";
+
+export const text2Markdown = (text: string) => {
+    if (!text) {
+        return ""
+    }
+    return (text as any)
+        .replaceAll('_', '\_')
+        .replaceAll('<i>', '_')
+        .replaceAll('</i>', '_')
+        .replaceAll('<b>', '__')
+        .replaceAll('</b>', '__')
+}
+
+class CopyUtils {
+
+    getOriginNode(node) {
```

Comment:
I don't see where this method is used. Please remove if it's not used.

Reply:
> I don't see where this method is used. Please remove if it's not used.

Ok, i just removed it